### PR TITLE
[MINOR] Fix the comment about usage of build.sh

### PR DESCRIPTION
**Pull request by caneGuy**
## Proposed changes

The usage of build.sh in the current comment is misleading. For example, if we execute with 'sh build.sh -clean', we will get the following error.
```[root@202cd6383716 incubator-doris]# sh build.sh -clean
Python 2.7.5
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: /usr/share/maven
Java version: 1.8.0_202, vendor: Oracle Corporation, runtime: /usr/java/jdk/jre
Default locale: en_US, platform encoding: ANSI_X3.4-1968
OS name: "linux", version: "4.19.76-linuxkit", arch: "amd64", family: "unix"
cmake version 3.12.3

CMake suite maintained and supported by Kitware (kitware.com/cmake).
build.sh: invalid option -- 'c'
build.sh: invalid option -- 'l'
build.sh: invalid option -- 'e'
build.sh: invalid option -- 'a'
build.sh: invalid option -- 'n'
```

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [x] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

### CODE REVIEW

**Comment by caneGuy** [comment]
cc @caiconghui

File: build.sh [hunk 1]
```diff
@@ -17,10 +17,16 @@
...
-# Usage:
-#    sh build.sh        build both Backend and Frontend.
-#    sh build.sh -clean clean previous output and build.
+# This script is used to compile Apache Doris(incubating).
+# Usage: 
+#    sh build.sh --help
+# Eg:
+#    sh build.sh                        build all
+#    sh build.sh  --be                  build Backend without clean
+#    sh build.sh  --fe --clean          clean and build Frontend and Spark Dpp application
```
Comment (morningman) [inline build.sh:26]:
```suggestion
#    sh build.sh  --fe --clean          clean and build Frontend and Spark Dpp application, without web UI.
```

Comment (morningman) [inline build.sh:26]:
```suggestion
#    sh build.sh  --fe --clean          clean and build Frontend and Spark Dpp application, without web UI.
```

File: build.sh [hunk 2]
```diff
@@ -17,10 +17,16 @@
...
+#    sh build.sh  --fe --be --clean     clean and build Frontend, Spark Dpp application and Backend
```
Comment (morningman) [inline build.sh:27]:
```suggestion
#    sh build.sh  --fe --be --clean     clean and build Frontend, Spark Dpp application and Backend, without web UI.
```

File: build.sh [hunk 3]
```diff
@@ -17,10 +17,16 @@
...
+#    sh build.sh  --spark-dpp           build Spark DPP application alone
+#    sh build.sh  --fe --ui             build Frontend web ui with npm
```
Comment (morningman) [inline build.sh:29]: Please add a new entry:
# sh build.sh  --fe --be --ui --clean     clean and build Frontend, Spark Dpp application, Backend and web UI

**Review by morningman — COMMENTED** [review]
Thank you for your PR.

**Comment by caneGuy** [comment]
Thanks for your time @morningman PTAL

**Review by caiconghui — APPROVED** [review]
LGTM

**Pull request merged**